packages/entity-browser/src/routes/detail/components/SubGrid/SubGrid.js

```diff
@@ -19,11 +24,17 @@ const SubGrid = props => {
 }
 
 SubGrid.propTypes = {
+  entityKey: React.PropTypes.string,
   formDefinition: React.PropTypes.shape({
     children: React.PropTypes.array
   }).isRequired,
   relationName: React.PropTypes.string.isRequired,
-  modelField: React.PropTypes.shape({targetEntity: React.PropTypes.string}).isRequired
+  modelField: React.PropTypes.shape(
+    {
+      targetEntity: React.PropTypes.string,
+      reverseRelationName: React.PropTypes.string
+    }
+  ).isRequired
```

Comment:
(the opening parenthesis `{` is on the same line as `shape(` everywhere else (the same applies to the closing parenthesis))